.gitignore

```diff
@@ -4,3 +4,4 @@ node_modules/
 *.txt
 *.json
```

Comment:
actually, just caught this one too. that's bad because it basically ignores `package.json` which doesn't get ignored in practice because it's already commited.

can you please remove `*.json` and also add `*.iml`?
thanks!